editor/editor_data.cpp

```diff
@@ -1042,23 +1042,22 @@ void EditorData::script_class_set_name(const String &p_path, const StringName &p
 }
 
 void EditorData::script_class_save_icon_paths() {
```

Comment:
This is no longer only saving icon paths, so it should be renamed to better describe what it does.